promoted_python_delivery_client/client/sdk_delivery.py

```diff
@@ -40,7 +40,7 @@ def run_delivery(self, request: DeliveryRequest) -> Response:
             index = index + 1
             offset = offset + 1
 
-        return Response(insertion=insertion_page)
+        return Response(insertion=insertion_page, request_id=req.request_id)
```

Comment:
@danbosnichill Do you think it makes sense to just leave this field blank for? Guessing there's nothing that can be done with it anyway, and could be confusing if the fallback isn't clearly indicated any other way

Reply:
Keep it like this.  Think of like an `execution_id`.  This function sets it on `req.request_id = str(uuid.uuid4())`.  The SDK produces an SDK DeliveryLog that appears in our tables.

Reply:
Ah, I saw where it set request_id but didn't realize it still produced a log entry